package/config/trustedanalytics-tar/application.conf

```diff
@@ -31,16 +31,28 @@ trustedanalytics.atk {
   api.port = ${PORT}
   api.request-timeout = 60s
 
+  #jdbc connector section
```

Comment:
We should have all of these in interfaces/reference.conf instead of over-riding it here.